greenboot.spec

```diff
@@ -41,99 +41,122 @@ Requires:           rpm-ostree
 Requires:           pam >= 1.4.0
 # While not strictly necessary to generate the motd, the main use-case of this package is to display it on SSH login
 Recommends:         openssh
-Provides:           greenboot-auto-update-fallback
```

Comment:
We probably want to preserve these to ensure a proper upgrade path from older releases

Reply:
will tackle in #124